plugin/src/main/java/io/jenkins/plugins/casc/Blacklist.java

```diff
@@ -0,0 +1,36 @@
+package io.jenkins.plugins.casc;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+
+@Restricted(NoExternalUse.class)
+class Blacklist {
+
+    private static final Logger LOGGER = Logger.getLogger(Blacklist.class.getName());
+
+    /**
+     * Syntax is: attribute-name space attribute-return-type
+     * <p>
+     * Enable FINE logging for this class to see all potential candidates
+     */
+    private static final List<String> BLACKLIST = Collections.singletonList(
+            "defaultProperties hudson.tools.ToolProperty"
```

Comment:
Could this currently blacklisted item be moved to the blacklist? https://github.com/jenkinsci/configuration-as-code-plugin/blob/c0dfdfbca098ce0fe5579f8069e7688030a57809/plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/GlobalConfigurationCategoryConfigurator.java#L34

Reply:
Is there test coverage for it?

Reply:
Yes, kinda :sweat: if the item being exported you should see something like this 
```
? 
: credential
```
in the export

Reply:
I'll take a look

Reply:
That doesn't turn out as a real attribute...
It doesn't show up anyway

```
restricted io.jenkins.plugins.casc.ConfigurationContext$Restriction
deprecated io.jenkins.plugins.casc.ConfigurationContext$Deprecation
version io.jenkins.plugins.casc.ConfigurationContext$Version
unknown io.jenkins.plugins.casc.ConfigurationContext$Unknown
nodeName java.lang.String
authorizationStrategy hudson.security.AuthorizationStrategy
systemMessage java.lang.String
agentProtocols java.lang.String
clouds hudson.slaves.Cloud
primaryView hudson.model.View
nodeProperties hudson.slaves.NodeProperty
mode hudson.model.Node$Mode
numExecutors int
quietPeriod java.lang.Integer
labelString java.lang.String
scmCheckoutRetryCount int
projectNamingStrategy jenkins.model.ProjectNamingStrategy
views hudson.model.View
crumbIssuer hudson.security.csrf.CrumbIssuer
markupFormatter hudson.markup.MarkupFormatter
disableRememberMe boolean
globalNodeProperties hudson.slaves.NodeProperty
slaveAgentPort int
securityRealm hudson.security.SecurityRealm
myViewsTabBar hudson.views.MyViewsTabBar
updateCenter hudson.model.UpdateCenter
viewsTabBar hudson.views.ViewsTabBar
proxy hudson.ProxyConfiguration
nodes hudson.model.Node
remotingSecurity jenkins.security.s2m.AdminWhitelistRule
name java.lang.String
properties hudson.model.ViewProperty
enabled java.lang.Boolean
captchaSupport hudson.security.captcha.CaptchaSupport
allowsSignup boolean
enableCaptcha boolean
users io.jenkins.plugins.casc.core.HudsonPrivateSecurityRealmConfigurator$UserWithPassword
password java.lang.String
id java.lang.String
password java.lang.String
id java.lang.String
sites hudson.model.UpdateSite
id java.lang.String
url java.lang.String
name java.lang.String
properties hudson.model.ViewProperty
typeFilter com.cloudbees.plugins.credentials.CredentialsTypeFilter
restrictions com.cloudbees.plugins.credentials.CredentialsProviderTypeRestriction
providerFilter com.cloudbees.plugins.credentials.CredentialsProviderFilter
tokenGenerationOnCreationEnabled boolean
creationOfLegacyTokenEnabled boolean
usageStatisticsEnabled boolean
enabled boolean
useBrowser boolean
authenticators jenkins.security.QueueItemAuthenticator
port int
artifactManagerFactories jenkins.model.ArtifactManagerFactory
configurationPath java.lang.String
shell java.lang.String
adminAddress java.lang.String
url java.lang.String
pollingThreadCount int
settingsProvider jenkins.mvn.SettingsProvider
globalSettingsProvider jenkins.mvn.GlobalSettingsProvider
defaultProperties hudson.tools.ToolProperty
installations hudson.model.JDK
defaultProperties hudson.tools.ToolProperty
installations hudson.tasks.Maven$MavenInstallation

```